client/ayon_core/hosts/hiero/api/plugin.py

```diff
@@ -604,31 +609,42 @@ def load(self):
         return track_item
 
 
-class Creator(LegacyCreator):
+class Creator(NewCreator):
```

Comment:
I would recommend to use `HieroCretor(Creator):`, and use just `from ayon_core.pipeline import Creator` import.